backend/main.go

```diff
@@ -40,11 +45,15 @@ func main() {
 	router := sw.NewRouter()
 
 	corsHandler := handlers.CORS(
-		handlers.AllowedOrigins([]string{"*"}), // Otwiera na wszystkie domeny
+		handlers.AllowedOrigins([]string{"*"}),
 		handlers.AllowedMethods([]string{"GET", "POST", "PUT", "DELETE", "OPTIONS"}),
 		handlers.AllowedHeaders([]string{"Authorization", "Content-Type"}),
 	)
+
 	health.ServiceStatus.MarkAsUp()
-	log.Printf("marking application readiness as UP")
-	log.Fatal(http.ListenAndServe(":8080", corsHandler(router)))
+	log.Printf("Marking application readiness as UP")
```

Comment:
Delete these - duplicate

Reply:
Chyba nie wiem gdzie - liveness !- readiness jeżeli to o to chodzi

Reply:
a ok mb